warehouse/models/mart/gtfs_quality/fct_monthly_route_id_changes.sql

```diff
@@ -0,0 +1,83 @@
+WITH reports_index AS (
+    SELECT * FROM {{ ref('idx_monthly_reports_site') }}
+),
+
+organization_dataset_map AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__organization_dataset_map') }}
+    WHERE reports_site_assessed
+),
+
+dim_routes AS (
+    SELECT  * FROM {{ ref('dim_routes') }}
+),
+
+month_start AS (
+    SELECT
+
+        reports_index.organization_name,
+        reports_index.organization_itp_id,
+        reports_index.publish_date,
+        reports_index.organization_source_record_id,
+
+        orgs.base64_url,
+        orgs.schedule_feed_key AS feed_key,
+
+        dim_routes.route_id
+
+    FROM reports_index
+    LEFT JOIN organization_dataset_map AS orgs ON (reports_index.date_start = orgs.date)
+        AND (reports_index.organization_source_record_id = orgs.organization_source_record_id)
+    INNER JOIN dim_routes ON (orgs.schedule_feed_key = dim_routes.feed_key)
+        WHERE dim_routes.route_id IS NOT NULL
+),
+
+month_end AS (
+    SELECT
+
+        reports_index.organization_name,
+        reports_index.organization_itp_id,
+        reports_index.publish_date,
+        reports_index.organization_source_record_id,
+
+        orgs.base64_url,
+        orgs.schedule_feed_key AS feed_key,
+
+        dim_routes.route_id
+
+    FROM reports_index
+    LEFT JOIN organization_dataset_map AS orgs ON (reports_index.date_end = orgs.date)
+        AND (reports_index.organization_source_record_id = orgs.organization_source_record_id)
+    INNER JOIN dim_routes ON (orgs.schedule_feed_key = dim_routes.feed_key)
+        WHERE dim_routes.route_id IS NOT NULL
+),
+
+month_comparison AS (
+    SELECT
+        t1.* EXCEPT (route_id),
+        t1.route_id AS start_table_source_id,
+        t2.route_id AS stop_table_source_id,
+        CASE
+            WHEN t2.route_id IS NULL THEN 'Removed'
+            WHEN t1.route_id IS NULL THEN 'Added'
```

Comment:
So, this looks right to me but somehow it doesn't seem to be getting the `added` rows for LA Metro last month 🤔 
![image](https://user-images.githubusercontent.com/55149902/216124153-d61a9fa6-553e-4ce7-b83a-536b44c3886a.png)